devicetypes/samsungsds/samsung-smart-doorlock.src/samsung-smart-doorlock.groovy

```diff
@@ -366,7 +364,7 @@ private def parseCommandResponse(String description) {
 				responseMap = [ name: "tamper", value: "detected", descriptionText: "Door forced open under door locked condition", isStateChange: true ]
 				break
 			case 7: // Door opened for over 30 seconds
-				responseMap = [ name: "lock", value: "autolockError", descriptionText: "Door remains opened for over 30 seconds" ]
+				responseMap = [ name: "tamper", value: "detected", descriptionText: "Door remains opened for over 30 seconds", isStateChange: true ]
```

Comment:
There doesn't seem to be anything that will clear `tamper` once set. Is that what you intended?

Reply:
It's fine if this isn't addressed in this PR since it's not a new issue but I do recommend that you figure out a way to clear the tamper, either after a delay or when the user takes a specific action